### ISSUE TITLE
Could not override model for lyssa

### ISSUE THREAD

**Issue by GreatestFool** [P3pp3rF1y/Reliquary#363]
[Error](https://pastebin.com/y1zVxcDq). And [entire log](https://paste.ee/p/x12G9), didn't fit on pastebin so it only lasts for a year. Here's [the modlist as well](https://pastebin.com/RAcnrdA8).

Not sure why it happens, but thought I'd mention it. Happens during boot.

**Comment by P3pp3rF1y** [comment]
what I can tell from logs seems like something is seriously broken there judging by all dangerous prefixes. Also the rod of lyssa has the model implemented the same way as vanilla fishing rod. I am pretty sure you won’t get this issue if you try to run just Reliquary by itself.

**Comment by GreatestFool** [comment]
I'll try that later. But I'm pretty certain by this point that something has irreparably gone wrong on my windows system. I'll try to see if I can recreate it using the same pack on linux later, hopefully tomorrow if I can make it. And I'll try it with only Reliquary before I do that, just in case, though I wouldn't be surprised if the issue persists.

**Comment by GreatestFool** [comment]
[Here's an error that happens with only reliquary in the pack.](https://pastebin.com/36U4E9h2) I've tried booting ATM3 on my main computer and my shitty laptop, both have the same issue with mentioning a lot of dangerous prefixes. I'm not sure why. I'll invade a friend of mine today or tomorrow and try to see if it is the same on his computer.

**Comment by P3pp3rF1y** [comment]
So based on that error I would say that Reliquary isn't the only thing in the pack. Reason being that nowhere in Reliquary nor in Minecraft/Forge there's defined any such thing as item_material. So it needs to come from somewhere outside of these.

**Comment by P3pp3rF1y** [comment]
Looking at this once more I am guessing you got your configs from some other pack where custom recipes for Reliquary were defined which crash it like this. Just delete your configs and you should be good.

**Comment by GreatestFool** [comment]
Ah, sorry for late reply got caught up in other things. The configs are not from another pack, I've added everything myself. I have not modified another pack to create mine, that'd be way to annoying considering I wouldn't know what has been changed and what hasn't.

I'm pretty sure the issue is simply caused by corruption of some kind, or something else. What I can't say. That's why I tried booting ATM3 to see, had the problems been absent there I could have safely said it was only my computer causing the issue. Since they were, even on different computers, well. I don't know what it means really.

The issue might as well stay closed, I have no idea what's causing it. And am trying to rebuild in linux and digging through a lot more than I prefer, but until now I haven't encountered any issues. Should a similar issue pop up around the time I get to Reliquary I'll just post the log here. If that's alright.

Either way, thanks for the help. I appreciate it.
